assets/src/components/propertiesPanel/header.tsx

```diff
@@ -20,16 +20,13 @@ import { Ghost, Vehicle, VehicleInScheduledService } from "../../realtime"
 import { RouteVariantName } from "../routeVariantName"
 import VehicleIcon, { Size, vehicleOrientation } from "../vehicleIcon"
 import TabList from "./tabList"
-import { TabMode } from "./tabPanels"
 import { currentRouteTab } from "../../models/routeTab"
 import ViewHeader from "../viewHeader"
 import { usePanelStateFromStateDispatchContext } from "../../hooks/usePanelState"
+import { IndividualPropertiesPanelProps } from "../propertiesPanel"
 
-interface Props {
+interface Props extends IndividualPropertiesPanelProps {
```

Comment:
**suggestion (non-blocking):** Extending `IndividualPropertiesPanelProps` feels like the wrong "level" to be working at because the properties panel as a whole could have extra stuff not related to the header in the future. Instead, we could consider having some third type (unsure of the name) that captures just the shared properties that would be relevant and `IndividualPropertiesPanelProps` could (for now at least) just be equal to that type.

Reply:
what do you think of 6abf46e3?